OrbitCore/EventBuffer.h

```diff
@@ -15,43 +15,52 @@
 
 class EventBuffer {
  public:
-  EventBuffer() : m_MaxTime(0), m_MinTime(LLONG_MAX) {}
+  EventBuffer() : min_time_(0), max_time_(LLONG_MAX) {}
 
   void Reset() {
     callstack_events_.clear();
-    m_MinTime = LLONG_MAX;
-    m_MaxTime = 0;
+    max_time_ = LLONG_MAX;
+    min_time_ = 0;
```

Comment:
The order here was reversed, max_time_ should be intialized to 0 and min_time_ to LLONG_MAX.

Reply:
We could also use std::numeric_limits<uint64_t>::max(); and std::numeric_limits<uint64_t>::min(); for consistency while we are at it.

Reply:
Thanks. Used std::numeric_limits<uint64_t>::max().